### PR TITLE
Fixed the EMS involved and now it search for EMS too

### PR DESCRIPTION
**Pull request by Annalouu**

### CODE REVIEW

File: ui/dashboard.html [hunk 1]
```diff
@@ -617,7 +617,7 @@
                 </div>
                 <div class="manage-officers-tags-container">
                     <div style="display: flex; align-items: center;">
-                        <div class="tags-title">Officers Involved</div>
+                        <div class="tags-title" id="bolos-officers-involved-tag-tittle">Officers Involved</div>
```
Comment (BackSH00TER) [inline ui/dashboard.html]: nit typo `title`

Comment (Annalouu) [reply ui/dashboard.html:620]: Idk what u mean by that, but what i did is tried on my local and recorded

Comment (BackSH00TER) [reply ui/dashboard.html]: You misspelled the world `title` as `tittle`. This is a nitpick comment that you should spell it correctly. 
```suggestion
                        <div class="tags-title" id="bolos-officers-involved-tag-title">Officers Involved</div>

```

Comment (BackSH00TER) [reply ui/dashboard.html]: Then you will need to update the other areas using this id

**Pull request closed without merging**